releasenotes/notes/0.12/use_conan_1.x-f12570e2cfc8bb26.yaml

```diff
@@ -0,0 +1,9 @@
+---
+issues:
+  - |
+    This release of Qiskit Aer is not compatible with the Conan 2.X release
+    series. If you are building Qiskit Aer from source manually ensure that
+    you are using a Conan 1.x release. Compatibility with newer versions
+    of Conan will be fixed in a future release. You can refer to
+    issue `#1730 <https://github.com/Qiskit/qiskit-aer/issues/1730>`__ for
+    more details.
```

Comment:
Don't we manually set this in our build files?  I don't think it should be possible to get into a situation where you're using the wrong Conan anymore, unless you're trying to manually manage things in a direct `setuptools` build, which is pretty much the wild west of build processes.

Reply:
We do, but it's still probably best to document it as a known limitation just in case it comes up.